widget/embedded/src/Widget.tsx

```diff
@@ -8,6 +8,7 @@ import React, { useContext, useEffect, useMemo, useState } from 'react';
 import { AppRouter } from './components/AppRouter';
 import { AppRoutes } from './components/AppRoutes';
 import { WidgetEvents } from './components/WidgetEvents';
+import { RANGO_BASE_URL } from './constants';
 import { globalFont } from './globalStyles';
 import { useLanguage } from './hooks/useLanguage';
 import { useTheme } from './hooks/useTheme';
```

Comment:
needReloadTheApp is one of the props related to ExperimentalFeatures located in the Dapp and if true, the app should be reloaded if the URL changes, and if false, no reload is required (not implemented in the widget).

Reply:
yeah but I think It should be handled in dapp not here. In dapp we reload the page and that's why the widget is reloaded as well.